### DISCUSSION TITLE
Pick coordinates of a 3D point

### DISCUSSION THREAD

**Discussion by clemencestr** [iTowns/itowns#1732] [Q&A] [answered]
Hello,

We want to select a point in our points cloud and display its coordinates. We are using view.pickObjectsAt but it always returns the coordinates of the current bounding box but not the coordinates of the point.
We also tried with pointCloudLayer.pickObjectsAt: the same problem occurs.

let objectsPicked = potreeLayer.pickObjectsAt(view, event, 0);
let objectsPicked = view.pickObjectsAt(event, 0, potreeLayer);

Does anyone have any idea why ?

Thanks

**Comment by gchoqueux** [comment, marked as answer]
Hello

yes, it's error in example, i fix the issue in  #1746
